_utils.sh

```diff
@@ -68,4 +68,15 @@ function set_pom_version {
   new_version=$1
 
   ${MVN} org.codehaus.mojo:versions-maven-plugin:2.8.1:set -DnewVersion=${new_version} -DgenerateBackupPoms=false --quiet
+}
+
+function is_flink_version_set_in_pom {
+  set +u
+  version=$(${MVN} help:evaluate -Dexpression="flink.version" -q -DforceStdout)
+  if [ -n "${version}" ]; then
```

Comment:
This isn't quite reliable because it fails if the pom doesn't contain a reference to `flink.version` at all (e.g. the Flink repo).
There it prints `null object or invalid expression`.
It works for the parent pom because it does define an empty `flink.version` property, mostly for documentation purposes.
Would be good to harden this so we don't fall into a trap in the future.

Reply:
Pushed an improvement of is_flink_version_set_in_pom function for undefined flink.version case. PTAL

Reply:
I forgot to tell that I tested the last change like this:
- did a connector-parent local release as is in parent_pom branch
- did a connector-parent local release with flink.version references removed from the pom
- did a cassandra connector local release

Reply:
> the script is made for connectors releases only

Sort of. The connectors were an opportunity to start a new collection of release scripts, and not carry forward issues that exist in the Flink ones.
The goal is definitely to use these for Flink at some point too. In some ways the connectors are just a test bed.

Reply:
> The goal is definitely to use these for Flink at some point too. In some ways the connectors are just a test bed.

Good to know ! 
Thanks for this information